src/Controllers/HomeController.cs

```diff
@@ -1,40 +1,48 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using System.Diagnostics;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Localization;
 using TrainingNet.Models;
 
 namespace TrainingNet.Controllers
 {
     [Route("")]
     public class HomeController : Controller
     {
+        private readonly IHtmlLocalizer<HomeController> _localizer;
+
+        public HomeController(IHtmlLocalizer<HomeController> localizer)
+        {
+            this._localizer = localizer;
+        }
+
         [HttpGet("")]
         public IActionResult Index()
-        {
+        {         
+            ViewData["Message"] = _localizer["HomePage"].Value;

```

Comment:
Same here as the previous file, try using a property instead of directly using the attribute

Reply:
Dont forget to change this to use the localizer as a property (like with the other controller)